src/ai/backend/manager/api/scaling_group.py

```diff
@@ -93,13 +94,24 @@ async def get_wsproxy_version(request: web.Request, params: Any) -> web.Response
                 if not wsproxy_addr:
                     wsproxy_version = "v1"
                 else:
-                    wsproxy_status = await query_wsproxy_status(wsproxy_addr)
-                    wsproxy_version = wsproxy_status["api_version"]
-                return web.json_response(
-                    {
-                        "wsproxy_version": wsproxy_version,
-                    }
-                )
+                    try:
+                        wsproxy_status = await query_wsproxy_status(wsproxy_addr)
+                        wsproxy_version = wsproxy_status["api_version"]
+                        return web.json_response(
+                            {
+                                "wsproxy_version": wsproxy_version,
+                            }
+                        )
+                    except aiohttp.ClientConnectorError:
+                        return web.HTTPInternalServerError(
+                            text=json.dumps(
+                                {
+                                    "type": "https://api.backend.ai/probs/internal-server-error",
+                                    "title": "Wsproxy of scaling group is invalid.",
+                                }
+                            ),
+                            content_type="application/problem+json",
```

Comment:
wow, could you give me any reference about this? seems interesting.

Reply:
Thanks for reviewing!
I just saw this code in repo.
https://github.com/lablup/backend.ai/blob/main/src/ai/backend/web/proxy.py#L262-#L270

And in Web UI,
https://github.com/lablup/backend.ai-webui/blob/main/src/lib/backend.ai-client-esm.ts#L290-#L479
This code receive status code.

Reply:
Please use `ai.backend.manager.api.exceptions.URLNotFound` instead of manually creating `web.HTTPNotFound` with the RFC78007-style data.

Reply:
Please use `ai.backend.manager.api.exceptions.URLNotFound` instead of manually creating `web.HTTPNotFound` with the RFC78007-style data.

Reply:
Thanks for reviewing!
I will fix it.